src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
Just went a flew a few packs with the 3d on a switch mode.  Did some intentional upsidedown crashes and was able to flip over and take off.  Got some video that I'm uploading...

Also used it for 3d mode in the air.  Worked pretty well and got used to it far faster then using center throttle method.  

I'm not a fan as much as using the mixer to do this.   `BOX3DDISABLESWITCH` could have been done entirely in the mixer as well.  Maybe a separate feature could be added: `FEATURE_3D_SWITCH`  Then a `BOX3DDISABLESWITCH` box (maybe renamed) could be used to select if ON then it is positive only.  If off it is negative only.

Really the bigger problem is running out of the 32 boxes in the first place.

Also the 3d mode in positive only seems really swimmy.  Not sure what the issue is there...

Thoughts?